packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/billing/groupRepayments/translations/Messages_fr_FR.json

```diff
@@ -10,7 +10,7 @@
   "telephony_group_billing_group_repayments_detail_total_call_singular": "{{ call }} Appel",
   "telephony_group_billing_group_repayments_detail_total_call_plural": "{{ call }} Appels",
   "telephony_group_billing_group_repayments_detail_total_call_empty": "Aucun Appel",
-  "telephony_group_billing_group_repayments_detail_total_duration_price": "({{ totalDuration }}sec.) soit {{ totalPrice }} HT",
+  "telephony_group_billing_group_repayments_detail_total_duration_price": "({{ totalDuration }}sec.) soit {{ totalPrice }} {{ currency }} HT",
```

Comment:
Shouldn't we display the price as a whole ? If we use french but our currency is `£` for instance it won't work

Reply:
You're right! We might handle it in a more global scope I think.
Especially in case where We will translate the whole Telecom Control Panel.

For now, I'll move forward regarding this Pull Request since it received a validation.